src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/DirectiveTypeDescriptorExtensions.cs

```diff
@@ -1,16 +1,15 @@
 using HotChocolate.Types;
+using static HotChocolate.ApolloFederation.WellKnownArgumentNames;
 
 namespace HotChocolate.ApolloFederation
```

Comment:
Internal extensions can stay in their namespace.

Reply:
Acknowledged